### ISSUE TITLE
No transcoding option? 

### ISSUE THREAD

**Issue by Anan5a** [jellyfin/jellyfin-androidtv#618]
**Describe the feature you'd like**	
	
Any information on if you plan to include the transcoding feature to the app? Like the one available in web ui and Android mobile? 
Streaming original media to my tv isn’t really viable or practical, because tv support max 1080p@h264 but many of my media is in 4k+hevc, right now playback blacks out, or player exit after few second of playback or buffers endlessly! 
**Additional context**	

Quality selection option is really necessary, 

Anyway I can help to implement that?

**Comment by MrChip53** [comment]
In the apps settings try setting your max bitrate to 15Mbps or 20Mbps.

**Comment by Anan5a** [comment]
@MrChip53  Dude I set it to 1Mbps and it still direct play 1080p@8Mbps (source)

**Comment by MrChip53** [comment]
> @MrChip53  Dude I set it to 1Mbps and it still direct play 1080p@8Mbps (source)

Make sure in the settings on the server the user account is allowed to transcode?

**Comment by Anan5a** [comment]
@MrChip53 yes, the account is allowed to transcode and max internet bitrate is set at 3Mbps

**Comment by MrChip53** [comment]
> @MrChip53 yes, the account is allowed to transcode and max internet bitrate is set at 3Mbps

Btw I just reread your issue and i think your problem is that your server hardware is incapable of transcoding the 4k to 1080p.

**Comment by Anan5a** [comment]
@MrChip53 nah, My server is Ryzen 7 3700X, 64GB ram, it's capable enough to transcode more than 1 4k stream

**Comment by BotBlake** [comment]
Hello there @Anan5a .
Is this issue still up-to-date?

Jellyfin Android TV is generally capable to tell the Server when to transcode.
If you have troubles with specific transcoding, please create a new issue with more details.

 
Kind regards.

**Comment by dadodasyra** [comment]
I do think that the issue also lies in the fact that jellyfin should have some kind of way to detect its own connection capability. like any streaming services right ?
